common.gradle

```diff
@@ -0,0 +1,107 @@
+apply plugin: 'application'
+apply plugin: 'java-library'
+
+sourceCompatibility = 8
+targetCompatibility = 8
+
+def rapidwrightDir = System.getenv("RAPIDWRIGHT_PATH") ? System.getenv("RAPIDWRIGHT_PATH") : file('.').toString()
+
+repositories {
+    if (project.hasProperty('useMavenLocal') && useMavenLocal == "True") {
+       project.logger.lifecycle('Using Maven Local Repository')
+       mavenLocal()
+    }
+    mavenCentral()
+}
+
+def os = System.getProperty("os.name").toLowerCase().contains("windows") ?
+                                       "win64-msvc2005x64" : "linux64-gcc"
+def sw = new StringWriter()
+def get_rapidwright_version_line = new File(rapidwrightDir + '/.github/workflows/build.yml').filterLine(sw) { 
+    line -> line.contains('RAPIDWRIGHT_VERSION:') 
+}
```

Comment:
Just curious, this code is duplicated from `build.gradle` however, in the other file, the file constructor changed from `new File()` to `file()`, but that same change was not propagated here.  Not sure the difference between the two.

Reply:
I cleaned up this duplication just now. My understanding is that `file()` is a Gradle provided function which is relative to the project directory, whereas `File` is I think relative to the current working directory. The former has useful when doing included builds.